Ginger/Ginger/UserControlsLib/UCTreeView/TreeViewItemGenericBase.cs

```diff
@@ -99,6 +102,11 @@ public void AddItemNodeBasicManipulationsOptions(ContextMenu CM, bool allowSave
                 TreeViewUtils.AddMenuItem(CM, "Open Containing Folder", OpenTreeItemFolderHandler, null, "@Folder_16x16.png");
                 mTreeView.AddToolbarTool("@Folder_16x16.png", "Open Containing Folder", OpenTreeItemFolderHandler);
             }
+            if (allowActionConversion)
+            {
```

Comment:
i asked on previous review to remove convertsion from the TreeViewItemGenericBase because it belong only to BusinessFlow tree items and not to all

Reply:
i kept it since it was required to handle in 2 classes (BusinessFlowTreeItem, BusinessFlowsFolderTreeItem), so added in "TreeViewItemGenericBase" also checked it was not displayed in other "TreeView". Now changed it and added in Business tree item classes.